README.md

```diff
@@ -1 +1,8 @@
-# mini-message-board
\ No newline at end of file
+# Mini Message Board
+
+## About the project
+This is the project mentioned in [The Odin Project](https://www.theodinproject.com/courses/nodejs/lessons/mini-message-board)
+
+- Buit with [express-generator](https://expressjs.com/en/starter/generator.html)
+
+- [pug](https://pugjs.org/api/getting-started.html) view template is used
```

Comment:
Add the details to set up and run the project to the README file.
For example: The code to run the server like `npm start`.